client/src/components/Form.js

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import isValidBrofile from '../lib/validatFunction';
+/*import { useLocalStorage } from '../hooks/useLocalStorage';*/
+
+export default function Form({ submitFunction, open }) {
+
+    const userBrofile = {
+        bro_username: '',
+        bro_name: '',
+        bro_location: '',
+        image: ''
+    }
+    const [newBrofile, setNewBrofile] = useState(userBrofile);
+    const [valid, setValid] = useState(false)
+
+    const handleChange = event => {
+        const field = event.target;
+        const value = field.value;
+        setNewBrofile({
+            ...newBrofile,
+            [field.name]: value
+        })
+
+        function submitForm(event) {
+            event.preventDefault();
+            if (isValidBrofile(newBrofile)) {
+                setValid(true);
+                submitFunction(newBrofile);
+                setNewBrofile(userBrofile);
+            }
+        }
+        return (
+            <>
+                <FormWrapper open={open} valid={valid} onSubmit={submitForm}>
+                    <input
+                        type='text'
+                        name='bro_username'
+                        placeholder='Enter your desired Username'
+                        onChange={handleChange}
+                        value={newBrofile.bro_username} />
+
+                    <input
+                        type='text'
+                        name='bro_name'
+                        placeholder='Enter your full name'
+                        onChange={handleChange}
+                        value={newBrofile.bro_name} />
+                    <input
+                        type='text'
+                        name='bro_location'
+                        placeholder='Enter your city'
+                        onChange={handleChange}
+                        value={newBrofile.bro_location} />
+
+                    <label>Add image
+        </label>
+                    <input
+                        type='file'
+                        name='image'
+                        placeholder='Add image'
+                        onChange={handleChange}
+                        value={newBrofile.image}
+                    />
+                    <CreateBrofileButton valid={valid} type='submit'> Create Profile</CreateBrofileButton>
```

Comment:
You could consider reusing your Default button component here instead of creating a new styled component